common/djangoapps/third_party_auth/tests/testutil.py

```diff
@@ -9,7 +9,9 @@
 
 import django.test
 import mock
+from mock import MagicMock, patch
```

Comment:
@felipemontoya I opted not modify original wildcard since in the tests before ours they are not using `patch` decorator, so they have not imported. Is it ok not modify it?